ci/container_scripts/install_extra_deps.sh

```diff
@@ -69,7 +69,7 @@ then
 fi
 
 # For debugging
-cargo --version
+rustup --version
```

Comment:
In #2688 I fix it so that `cargo --version` does work, and updated the comment. I think we do want to force installing the toolchain here.

It probably *does* make sense to run `rustup --version` here for debugging/logging purposes too though. So I guess whenever we resolve the conflict between the two we should keep both.